src/patternfly/components/Table/table-tree-view.scss

```diff
@@ -59,7 +60,14 @@ $pf-c-tree-view--MaxDepth: 10;
   }
 
   > .pf-c-table__check {
-    margin-right: var(--pf-c-table__tree-view-main--c-table__check--MarginRight);
```

Comment:
I think this will break core users who don't get the markup change. We should probably keep all the positioning stuff limited to the `<label>`, which is basically our opt-in for the change.